blog/2023-12-18-cloud-native-configuration-language-kcl/index.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: 2023-12-18-cloud-native-configuration-language-kcl
+title: Cloud Native Strategy Configuration Language - KCL
+authors:
+  name: KCL Team
+  title: KCL Team
+tags: [KCL, Meeting]
+---
+
+
+> This blog is a review of the content of the KCL language part of the speech at the 2023 CNCF KCD ShenZhen meeting by KusionStack leader Dayuan Li and KCL project Maintainer Zhe Zong. The main content of this article is a review of the content of the KCL language part of the speech, the activity link: [https://community.cncf.io/events/details/cncf-kcd-hangzhou-presents-kcd-shenzhen-2023/](https://community.cncf.io/events/details/cncf-kcd-shenzhen-presents-kcd-shenzhen-2023/)
+
+
+## Self-introduction
+
+Hello everyone, I am Zhe Zong from Ant Group, and also the Manitinor of the KCL language project. I am very honored to participate in this KCD Shenzhen conference. Next, I will introduce some content about the KCL language part.
+
+![intro-myself.png](/img/blog/2023-12-18-cloud-native-configuration-language-kcl/intro-myself.png)
+
+## In the cloud-native era, infrastructure as code (IaC) is the core of developer experience
+
+In today's rapidly developing technical world, infrastructure as code (IaC) has become the key to automating and managing cloud resources, and IaC has also become the core part of developer experience, bringing convenience and efficiency, but it also brings a series of challenges.
+
+![intro-iac.png](/img/blog/2023-12-18-cloud-native-configuration-language-kcl/intro-iac.png)
```

Comment:
It looks strange mixed with Chinese images on English blogs.

Reply:
fixed